pages/c/_cluster/explorer/index.vue

```diff
@@ -242,118 +456,86 @@ export default {
-          </h5>
-          <div v-if="gatekeeperEnabled">
-            <div class="row mt-10">
-              <SortableTable
-                :rows="filteredConstraints"
-                :headers="constraintHeaders"
-                :search="false"
-                :table-actions="false"
-                :row-actions="false"
-                key-field="id"
-                no-rows-key="clusterIndexPage.sections.gatekeeper.noRows"
-              />
-            </div>
-          </div>
-          <div v-else>
-            <hr class="mt-35 mb-10" />
-            <div class="mt-35 mb-35 text-center">
-              <div>
-                <t k="clusterIndexPage.sections.gatekeeper.disabled" />
-              </div>
-              <n-link
-                :to="{ name: 'c-cluster-gatekeeper' }"
-                role="link"
-                type="button"
-                class="btn role-link"
-              >
-                <t k="clusterIndexPage.sections.gatekeeper.buttonText" />
-              </n-link>
-            </div>
-          </div>
-        </InfoBox>
-      </div>
+    <Glance :provider="displayProvider" :kubernetes-version="cluster.kubernetesVersion" :total-nodes="(nodes || []).length" :created="cluster.metadata.creationTimestamp" />
+    <div class="resource-gauges">
+      <ResourceGauge v-for="resourceGauge in resourceGauges" :key="resourceGauge.name" v-bind="resourceGauge" />
+      <div v-for="(filler, i) in resourceGaugesFiller" :key="i" class="filler" />
     </div>
-    <div class="row">
-      <div class="col span-12">
-        <InfoBox>
-          <h5>
-            <t k="clusterIndexPage.sections.events.label" />
-          </h5>
-          <div class="row mt-10">
-            <SortableTable
-              :rows="events"
-              :headers="eventHeaders"
-              key-field="id"
-              :search="false"
-              :table-actions="false"
-              :row-actions="false"
-              :paging="true"
-              :rows-per-page="10"
-              default-sort-by="date"
-            />
-          </div>
-        </InfoBox>
-      </div>
+    <div v-if="nodes.length" class="hardware-resource-gauges">
+      <HardwareResourceGauge name="Pods Reserved" :total="podsReserved.total" :useful="podsReserved.useful" :suffix="t('clusterIndexPage.hardwareResourceGauge.podsReserved')" />
+      <HardwareResourceGauge name="Cores Reserved" :total="cpuReserved.total" :useful="cpuReserved.useful" :suffix="t('clusterIndexPage.hardwareResourceGauge.coresReserved')" />
+      <HardwareResourceGauge name="Ram Reserved" :total="ramReserved.total" :useful="ramReserved.useful" :units="ramReserved.units" :suffix="t('clusterIndexPage.hardwareResourceGauge.ramReserved')" />
     </div>
+    <div v-if="nodes.length" class="hardware-resource-gauges live">
+      <HardwareResourceGauge name="Cores Used" :total="cpuUsed.total" :useful="cpuUsed.useful" :suffix="t('clusterIndexPage.hardwareResourceGauge.coresUsed')" />
```

Comment:
Can't assume these are going to be present, some clusters do not have metrics server installed and will not have "used" data.

Reply:
You're right. It was supposed to check nodeMetrics.length but I messed that up.